CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -DDROP_CGAL ${CPPSTDFLAGS} -Wall -
 
 # Make sure we find the Find*.cmake functions distributed with this package
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
-set(CMAKE_PREFIX_PATH $ENV{PODIO} $ENV{FCCEDM})
+set(CMAKE_PREFIX_PATH $ENV{ROOTSYS} $ENV{PODIO} $ENV{FCCEDM})
```

Comment:
This entire line is not needed when using the fcc init script. May still be useful for compilation outside the lxplus environment.

The change should not be necessary, it's already added in [thisroot.sh](https://github.com/root-project/root/blob/master/config/thisroot.sh#L83)